Documentation/networking/overview.md

```diff
@@ -341,13 +341,27 @@ As a reminder, the sort order of the loaded networks is detailed in the chapter
 
 ### Socket Activation
 rkt also supports socket activation.
-This is documented in [Socket-activated service](../using-rkt-with-systemd.md#socket-activated-service).
+This is documented in [Socket-activated service][socket-activated].
 
 ## More Docs
 
 ##### Examples
-* [bridge plugin](examples-bridge.md)
+* [bridge plugin][examples-bridge]
 
 ##### Other topics:
-* [DNS configuration](dns.md)
-* [Overriding defaults](overriding-defaults.md)
+* [DNS configuration][dns]
+* [Overriding defaults][overriding]
+
+
+[cni]: https://github.com/appc/cni
+[cni-dhcp]: https://github.com/appc/cni/blob/master/Documentation/dhcp.md
+[cni-flannel]: https://github.com/appc/cni/blob/master/Documentation/flannel.md
+[dns]: dns.md
+[examples-bridge]: examples-bridge.md
+[ipvlan]: https://www.kernel.org/doc/Documentation/networking/ipvlan.txt
+[ipvlan-dupv6]: http://thread.gmane.org/gmane.linux.network/363346/focus=363345
```

Comment:
I wasn't sure what to do with this. Gmane says they're still bringing all their old content online, but gmane.linux.network isn't there yet and I wasn't able to find another source for this message. I'll just remove the link entirely, I think.